lib/node_modules/@stdlib/math/generics/statistics/incrprod/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@stdlib/math/generics/statistics/incrsum",
+  "version": "0.0.0",
+  "description": "Compute a product incrementally.",
+  "author": {
+    "name": "The Stdlib Authors",
+    "url": "https://github.com/stdlib-js/stdlib/graphs/contributors"
+  },
+  "contributors": [
+    {
+      "name": "The Stdlib Authors",
+      "url": "https://github.com/stdlib-js/stdlib/graphs/contributors"
+    }
+  ],
+  "scripts": {},
+  "main": "./lib",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/stdlib-js/stdlib.git"
+  },
+  "homepage": "https://github.com/stdlib-js/stdlib.git",
+  "keywords": [
+    "stdlib",
+    "stdmath",
+    "statistics",
+    "stats",
+    "mathematics",
+    "math",
+    "multiplication",
+		"mult",
```

Comment:
This one was annoying, atom showed the correct spacing but git(hub) knew better. Opening it in another editor fixed it.